app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -143,7 +153,8 @@
 
 %div{"ng-if" => defined?(tls_ca_certs_hide) ? false : true}
   .form-group{"ng-if"=> "emsCommonModel.emstype == 'rhevm' || " +                                          |
-                        "(emsCommonModel.ems_controller == 'ems_container' && " +                          |
+                        "((emsCommonModel.ems_controller == 'ems_container' || " +                         |
+                        "emsCommonModel.emstype == 'hawkular') && " +                               |
```

Comment:
This is perhaps too nested, hard to read.
One option that might be more readable if you put the custom condition first (untested):
```
                         "(emsCommonModel.#{prefix}_security_protocol == 'ssl-with-validation-custom-ca' &&" |
                         " (emsCommonModel.ems_controller == 'ems_container' || " +                          |
                         "  emsCommonModel.emstype == 'hawkular'))"}                                         |

However the meaning of security_protocol is provider-dependent and this file already does a lot of copy-pasting to keep different providers independent.
What do you think of doing separate (container && custom) || (hawkular && custom)?